### ISSUE TITLE
F12 on 'new' borks the language server

### ISSUE THREAD

**Issue by jamshark70** [scztt/LanguageServer.quark#19]
Not sure if this is a language server or vscode plugin issue.

I had a bit of code saying `Document.new(...)` and I wanted to jump to `new`. So I put the cursor on `new` and hit F12, and the blue progess thingy at the top of the window kept going and going and going...

At this point, the interpreter is OK (shift-return, no problem) but no language server interaction works anymore. Even "restart sclang" gives an error (stopping the server timed out).

F12 on a method name with fewer implementations (e.g. `sched` or `use`) works fine. So it may be a scalability problem.

**Comment by scztt** [comment]
Yes, this is *probably* just because there's a `new` method for every class, so it's trying to return a list of a few thousand. I need to rework this to use streams / routines to provide results, so that partial results can be sent.
